cmd/semdiffstat/main.go

```diff
@@ -49,22 +50,23 @@ func main() {
 		log.Fatalf("could not parse Go files %v and %v: %v\n", aName, bName, err)
 	}
 
+	w := new(tabwriter.Writer)
```

Comment:
You can do this in one line with `tabwriter.NewWriter`.

Reply:
Done